torch/export/unflatten.py

```diff
@@ -1026,32 +1048,47 @@ def _sink_params(
     # Also remove from call_module nodes
     call_module_nodes = filter(lambda n: n.op == "call_module", graph.nodes)
     for node in call_module_nodes:
-        node.args = tuple(filter(lambda n: n.name not in inputs_to_state, node.args))
+        submodule = _recursive_getattr(module, node.target.split("."))
+        if submodule is not None and id(submodule) in module_id_to_inputs_removed:
+            node.args = tuple(
```

Comment:
this is probably the most important line in this PR, so we should add a comment here. Maybe just something simple like "remove placeholder from call_module node arguments, only if we've erased the placeholder node in the corresponding _sink_params() call"

Reply:
Done, thanks!